frontend/src/views/Abfrage.vue

```diff
@@ -469,67 +481,76 @@ export default class Abfrage extends Mixins(
-          this.handleSuccess(dto);
+          this.handleSuccess(dto, showToast);
         });
       } else if (this.isEditableBySachbearbeitung()) {
         await this.patchAbfrageInBearbeitungSachbearbeitung(
           mapToInfrastrukturabfrageInBearbeitungSachbearbeitungDto(this.abfrage),
           this.abfrage.id as string,
           true
         ).then((dto) => {
-          this.handleSuccess(dto);
+          this.handleSuccess(dto, showToast);
         });
       }
     } else {
       this.showWarningInInformationList(validationMessage);
     }
   }
 
-  private handleSuccess(dto: InfrastrukturabfrageDto): void {
+  private handleSuccess(dto: InfrastrukturabfrageDto, showToast: boolean): void {
     this.saveAbfrageInStore(new InfrastrukturabfrageModel(dto));
     this.$store.dispatch("search/resetAbfrage");
     if (this.isNewAbfrage()) {
       this.$router.push({ path: "/abfragenuebersicht" });
-      Toaster.toast(`Die Abfrage wurde erfolgreich gespeichert`, Levels.SUCCESS);
+      if (showToast) Toaster.toast(`Die Abfrage wurde erfolgreich gespeichert`, Levels.SUCCESS);
     } else {
-      Toaster.toast(`Die Abfrage wurde erfolgreich aktualisiert`, Levels.SUCCESS);
+      if (showToast) Toaster.toast(`Die Abfrage wurde erfolgreich aktualisiert`, Levels.SUCCESS);
     }
   }
 
   private async startStatusUebergang(transition: TransitionDto) {
     if (!this.isDirty()) {
+      let toastMessage = "Die Abfrage hatte einen erfolgreichen Statuswechsel";
+      if (transition.url === "abfrage-schliessen") {
+        toastMessage = "Die Abfrage wird ohne Einbindung der Fachreferate abgeschlossen";
+      }
       const validationMessage: string | null = this.findFaultInInfrastrukturabfrageForSave(this.abfrage);
       if (_.isNil(validationMessage)) {
-        const requestSuccessful = await this.statusUebergangRequest(transition, this.abfrageId);
-        if (requestSuccessful) {
-          if (!(transition.buttonName === "IN BEARBEITUNG SETZEN")) {
-            this.returnToUebersicht("Die Abfrage hatte einen erfolgreichen Statuswechsel", Levels.SUCCESS);
+        const response = await this.statusUebergangRequest(transition, this.abfrageId, this.anmerkung);
+        if (response instanceof Response) {
+          if (response.ok) {
+            if (!(transition.url === "in-bearbeitung-setzen")) {
+              this.returnToUebersicht(toastMessage, Levels.SUCCESS);
+            } else {
+              this.setSelectedAbfrageInStore();
+              this.getTransitions(this.abfrageId, true).then((response) => {
+                this.possbileTransitions = response;
+              });
+            }
+            this.selectAbfrage();
           } else {
-            this.setSelectedAbfrageInStore();
-            this.getTransitions(this.abfrageId, true).then((response) => {
-              this.possbileTransitions = response;
-            });
+            this.anmerkung = "";
+            this.handleResponseNotOk(true, response);
```

Comment:
Nach Rücksprache mit Boxi ist uns (noch) nicht klar, wann der Fall respone != OK auftreten soll, da das Backend entweder eine Expection wirft, die in StatusUebergangApiRequestMixin im _catch_ abgefangen wird, oder Response.OK zurückliefert. Könnten wir das bitte gemeinsam klären und abstimmen. Aus jetziger Sicht könnten wir dann auf ErrorHandler.handleResponseNotOk() verzichten.